packages/flutter/test/material/icon_button_test.dart

```diff
@@ -275,11 +275,17 @@ void main() {
 
   testWidgets('Small icons comply with VisualDensity requirements', (WidgetTester tester) async {
     final bool material3 = theme.useMaterial3;
+    final ThemeData themeDataM2 = ThemeData(visualDensity: const VisualDensity(horizontal: 1, vertical: -1), useMaterial3: material3);
+    final ThemeData themeDataM3 = ThemeData(
```

Comment:
Could be indented a little more fluttery:
```dart
    final ThemeData themeDataM3 = ThemeData(
      useMaterial3: material3
      iconButtonTheme: IconButtonThemeData(
        style: IconButton.styleFrom(
          visualDensity: const VisualDensity(horizontal: 1, vertical: -1),
        ),
      ),
    );
```

Reply:
Thanks for the suggestions. Reformatted!